packages/generate-loader/templates/src/_index.js.tpl

```diff
@@ -3,9 +3,9 @@
  * https://webpack.js.org/contribute/writing-a-loader
  */
 
-export default function loader(source) {
+module.exports = function loader(source) {
```

Comment:
I think we should implement `--babel` option to this in future (put babel config and generate es module syntax)

Reply:
Will implement in future PR.